routing/router.go

```diff
@@ -168,8 +169,20 @@ func NewRouter(
 		authController = headerAuth
 	}
 
+	// Initialize graph cache and refresh job manager for per-session graph caching
+	graphCacheConfig := graph.LoadGraphCacheConfig(*conf)
+	graphCache := graph.NewGraphCache(context.Background(), graphCacheConfig)
+	refreshJobManager := graph.NewRefreshJobManager(context.Background())
```

Comment:
There's a root context that gets passed to other components in the root cmd. Ideally this would get passed in and used here.

Reply:
OK, I'll wire that up.